index.html

```diff
@@ -56,7 +56,19 @@
             publisher: "W3C",
             date: "08 May 2019",
           },
-        },
+	  JEITA_IT-4002: {
```

Comment:
Wrap by `'`.
Or, use `_` instead of `-`. (Syntax restriction of JavaScript on label)